### DISCUSSION TITLE
Why my project run using cypress-cloud are not shown on my sorry-cypress dashboard ?

### DISCUSSION THREAD

**Discussion by samixchoumi** [sorry-cypress/sorry-cypress#802] [Q&A] [answered]
Hello, I'm new to the use of sorry-cypress dashboard, i followed the tutorial here https://docs.sorry-cypress.dev/guide/dashboard-and-api and deployed the director, dashboard using kub here https://crumbhole.com/indroducing-the-sorry-cypress-helm-chart/

I've set the generated url from kub to the director export const `cloudServiceUrl = 'urldirector'` and it works fine. But when i go to the dashboard (it load perfectly) the project doesn't appear on it after the use of cypress-cloud like that : `npx cypress-cloud run --record --key yyy --parallel --ci-build-id $CI_PIPELINE_ID --env allure=true,configFile=dev --browser chrome`.

Maybe I missed something but I don't know what. Everything seem correct.

I've added manually the projectId on the dashboard by creating a project with the projectId I want and now i've 0 test run.

This is what I got when I launch the command (it's weird because i've 2 spec files) :   
```
currents:api network request: { method: 'post', url: '/runs', headers: { Accept: 'application/json, text/plain, */*', 'Content-Type': 'application/json', 'x-cypress-request-attempt': 0, 'x-cypress-version': '12.14.0', 'x-ccy-version': '1.7.4' }, data: { ci: { params: {}, provider: null }, specs: [ 'cypress/integration/test/test1.feature', 'cypress/integration/test/test2.feature' ], commit: { branch: 'master', remoteOrigin: 'git@gitlab.com:project/qat/automation-test.git', authorEmail: 'samixchoumi@project.com', authorName: 'samixchoumi', message: "Merge branch 'feature/add_cypres-cloud_and_cypress-api' into 'master'\n" + '\n' + 'Update package.json and add cy.api() usage\n' + '\n' + 'See merge request project/qat/automation-test!17', sha: 'e7896c94cc4ecfd27f55b5be6ce9ee80cddc87db' }, group: undefined, platform: { osName: 'darwin', osVersion: '22.5.0', osCpus: [Array], osMemory: [Object], browserName: 'Chrome', browserVersion: '114.0.5735.133' }, parallel: false, ciBuildId: '--env', projectId: 'teste2e', recordKey: 'yyy', specPattern: [ './cypress/integration/**/*.feature' ], tags: [], testingType: 'e2e', batchSize: 3, autoCancelAfterFailures: undefined } } +0ms
  currents:api network response: { status: 200, statusText: 'OK', headers: AxiosHeaders { 'x-powered-by': 'Express', 'content-type': 'application/json; charset=utf-8', 'content-length': '239', etag: 'W/"ef-KSFUTHeyp7ortaiM789H7mbePWs"', date: 'Fri, 23 Jun 2023 12:27:25 GMT', 'keep-alive': 'timeout=5', connection: 'close' }, data: { groupId: '--env', machineId: '0c9d5af4-ecf8-4c18-808a-c62b43abdd9f', runId: '2ee7db2c8c4567ac44b7bcab75a3d4a3', runUrl: 'sorry-cypress-dashboard.dev-itx5.itx5.dev/run/2ee7db2c8c4567ac44b7bcab75a3d4a3', isNewRun: false, warnings: [] } } +159ms
🎥 Run URL: sorry-cypress-dashboard.dev-itx5.itx5.dev/run/2ee7db2c8c4567ac44b7bcab75a3d4a3
  currents:capture resetting captured stdout +3s
  currents:api network request: { method: 'post', url: 'runs/2ee7db2c8c4567ac44b7bcab75a3d4a3/instances', headers: { Accept: 'application/json, text/plain, */*', 'Content-Type': 'application/json', 'x-cypress-request-attempt': 0, 'x-cypress-version': '12.14.0', 'x-ccy-version': '1.7.4', 'x-cypress-run-id': '2ee7db2c8c4567ac44b7bcab75a3d4a3' }, data: { runId: '2ee7db2c8c4567ac44b7bcab75a3d4a3', groupId: '--env', machineId: '0c9d5af4-ecf8-4c18-808a-c62b43abdd9f', platform: { osName: 'darwin', osVersion: '22.5.0', osCpus: [Array], osMemory: [Object], browserName: 'Chrome', browserVersion: '114.0.5735.133' } } } +5ms
  currents:api network response: { status: 200, statusText: 'OK', headers: AxiosHeaders { 'x-powered-by': 'Express', 'content-type': 'application/json; charset=utf-8', 'content-length': '71', etag: 'W/"47-NP1mIejCWhZPVkT9pyQygCAB7jQ"', date: 'Fri, 23 Jun 2023 12:27:25 GMT', 'keep-alive': 'timeout=5', connection: 'close' }, data: { spec: null, instanceId: null, claimedInstances: 2, totalInstances: 2 } } +126ms
  currents:runner No more tasks to run. Uploads queue: 0 +0ms

====================================================================================================


  Cloud Run Finished  

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │     Spec                                         Tests   Passing   Failing   Pending   Skipped │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔   No specs executed                    0ms         -         -         -         -         - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘

🏁 Recorded Run: sorry-cypress-dashboard.dev-itx5.itx5.dev/run/2ee7db2c8c4567ac44b7bcab75a3d4a3

```

EDIT : 
If it's a memory problem which parameter from https://github.com/sorry-cypress/charts/blob/main/charts/sorry-cypress/README.md should I update ?

EDIT 2 :
I found that when I change the projectId, the run work (using `cypress-cloud run` but I have nothing on the dashboard), then I Relaunch it and I have the same message (no specs executed), it's like I can make it work only 1 time for each project.

**Comment by samixchoumi** [comment]
Another question, when i'm on a run and I click on the link (for example : sorry-cypress-dashboard.dev-itx5.itx5.dev/run/518b03e92e7bb7b9614e178a589b3f07) it redirect me to this screen : 
![image](https://github.com/sorry-cypress/sorry-cypress/assets/22171928/ebb52c20-ba83-4762-bb07-5b148d2e0881)
Is that normal ? I've a project with the same projectId that i'm using, normally it should be in https://sorry-cypress-dashboard.dev-itx5.itx5.dev/teste2e/runs no ?

EDIT : 
And on the project, I've nothing too :
![image](https://github.com/sorry-cypress/sorry-cypress/assets/22171928/c802e01d-3006-4826-9570-e156aafb4061)

**Reply by agoldis** [reply, marked as answer]
Here are a few resources that can help you: 
https://currents.dev/readme/guides/cypress-ci-build-id
https://currents.dev/readme/integration-with-cypress/troubleshooting

**Reply by samixchoumi** [reply]
By switching to mongodb, it worked (and i misunderstood the ci-build-id option)
